### PR TITLE
feat: display torrent location + move torrent location

### LINKED ISSUE #10: m to move a torrent direcrory in torrents tab

### PR DESCRIPTION
**Pull request by aidanaden**
closes #10

includes displaying torrent location (makes it easier to tell if moving locations is successful)

### CODE REVIEW

File: rm-main/src/ui/tabs/torrents/tasks/move_torrent.rs
Comment (micielski) [inline rm-main/src/ui/tabs/torrents/tasks/move_torrent.rs:24]: use InputManager::new_with_value (or something like this), with the value of the torrent's old directory

File: rm-main/src/ui/tabs/torrents/tasks/move_torrent.rs
Comment (micielski) [inline rm-main/src/ui/tabs/torrents/tasks/move_torrent.rs:19]: change from to_move to torrents_to_move, since it will be already named like that in the struct

**Review by micielski — CHANGES_REQUESTED** [review]
Also, could we get rid of this directory column for the time being? For me every torrent is just in one default directory, and after I'll refactor the config then we can make it so that the headers will be configurable (the order and which to include)

**Comment by aidanaden** [comment]
> Also, could we get rid of this directory column for the time being? For me every torrent is just in one default directory, and after I'll refactor the config then we can make it so that the headers will be configurable (the order and which to include)

can we add this in the config? my torrents are all in separate folders based on their category - we can leave it disabled by default

**Comment by aidanaden** [comment]
i can hold off till your refactor before merging this PR though .. having directory displayed is a requirement for me LOL

**Pull request merged**
